src/BuildScriptGenerator.Common/ShellScriptBuilder.cs

```diff
@@ -158,11 +158,19 @@ public ShellScriptBuilder AddStringDoesNotExistInFileCheck(string searchString,
                 "exit 1; fi");
         }
 
+        /// <summary>
+        /// Append a command to the shell script that sets the ORYX_SDK_STORAGE_BASE_URL to the value
+        /// of ORYX_TEST_SDK_STORAGE_URL if ORYX_TEST_SDK_STORAGE_URL exists in the environment that is executing this code.
```

Comment:
nit: out of curiosity, any reason we introduce a new `ORYX_TEST_SDK_STORAGE_URL` environment variable rather than just setting the existing `ORYX_SDK_STORAGE_BASE_URL` one in the calling scripts? Is this just to explicitly differentiate what's being set for test execution and what is actually being used within the container?

Reply:
I couldn't think of a better way to essentially get the functionality of passing in a parameter to the tests

The reason that I went with introducing the `ORYX_TEST_SDK_STORAGE_URL` environment variable is so that we do not need to re-generate every image in order to change the storage account url used for testing.

It allows us to use DEV by default, which is what happens now. But if we want to use something other than DEV I had 2 solutions:
- remove the "AddDefaultTestEnvironmentVariables" from our tests and have it use the default that exists in the container (requires re-generation of images to pass in the new `ORYX_SDK_STORAGE_BASE_URL`)
- add a new env var that tests will use to determine the url to use (or default to dev)

Reply:
Yep makes sense to me, good call 👍